examples/mcts_example.py

```diff
@@ -125,6 +126,7 @@ def _extract_actions(self, game):
                     actions.append(Action(action_choice.action_type, position=position))
             else:
                 actions.append(Action(action_choice.action_type))
+        random.shuffle(actions)
```

Comment:
Can you make this optional and disabled by default? I want the script to follow the tutorial.